server/game/effects.js

```diff
@@ -497,13 +497,15 @@ const Effects = {
             Effects.modifyStrength(value)
         ];
     },
-    blank: {
-        apply: function(card) {
-            card.setBlank();
-        },
-        unapply: function(card) {
-            card.clearBlank();
-        }
+    blank: function(includingTraits = false) {
```

Comment:
As above, `.blank(true)` doesn't read very well. Either a separate effect name (e.g. `.blankIncludingTraits()` can use the same underlying implementation) or sending a human readable flag (`.blank({ includeTraits: true })`) may be better.